app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -273,36 +272,40 @@ fun CommentOrPostNodeHeader(
                     bottom = MEDIUM_PADDING,
                 ),
     ) {
-        Row(
-            horizontalArrangement = Arrangement.spacedBy(SMALL_PADDING),
-            verticalAlignment = Alignment.CenterVertically,
-        ) {
-            if (deleted) {
-                Icon(
-                    imageVector = Icons.Outlined.Delete,
-                    contentDescription = stringResource(R.string.commentOrPostHeader_deleted),
-                    tint = MaterialTheme.colorScheme.error,
-                )
-                DotSpacer(style = MaterialTheme.typography.bodyMedium)
-            }
-
-            PersonProfileLink(
-                person = creator,
-                onClick = { onPersonClick(creator.id) },
-                showTags = true,
-                isPostCreator = isPostCreator,
-                isDistinguished = isDistinguished,
-                isCommunityBanned = isCommunityBanned,
-                showAvatar = showAvatar,
+        val centerMod = Modifier.align(Alignment.CenterVertically)
```

Comment:
This centerMod is necessary on flowRow unfortunately. I wish it could do some cross-axis alignment, but it doesn't seem to work.